Tests/ML/test_data/cxr_test_dataset/Data_Entry_2017.csv

```diff
@@ -0,0 +1,201 @@
+Image Index
+1.dcm
```

Comment:
What's this file? Is this auto-generated by a test, and you added it?

Reply:
Auto-generated. I've fixed the .gitignore